pkgs/by-name/in/intiface-central/package.nix

```
@@ -29,8 +32,9 @@ flutterPackages.stable.buildFlutterApplication rec {
     name = "${pname}-${version}-cargo-deps";
     inherit src;
     sourceRoot = "${src.name}/intiface-engine-flutter-bridge";
-    hash = "sha256-1j1maTrR7V/Tlu7uimTJuoEtxhn5Hn0VmR0GnJuKIoo=";
+    hash = "sha256-2Sk0Ww9lawWke3xRToghed7SaJTeCDEQlXUy989oiTA=";
```

This has seems to be incorrect
```suggestion
    hash = "sha256-z9i0xT0e8G9spOmXpI2yFee/y3ZEh69YqL+7zRWszzA=";
   ```